src/PrestaShopBundle/Form/Admin/Type/ShopRestrictionCheckboxType.php

```diff
@@ -36,6 +35,9 @@
 use Symfony\Component\Form\FormView;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
+/**
+ * {@inheritdoc}
```

Comment:
it's better to write a more specific doc, explaining the responsibility of this class